frontend/src/components/common/Resource/EditorDialog.tsx

```diff
@@ -210,34 +214,43 @@ export default function EditorDialog(props: EditorDialogProps) {
       return;
     }
 
-    const { obj: codeObj } = getObjectFromCode(code);
-
-    const { kind, apiVersion } = (codeObj || {}) as KubeObjectInterface;
-    if (codeObj === null || (!!kind && !!apiVersion)) {
-      setDocSpecs({
-        error: codeObj === null,
-        kind,
-        apiVersion,
-      });
+    const { obj: codeObjs } = getObjectsFromCode(code);
+    if (codeObjs && codeObjs?.length > 0) {
+      if(!Array.isArray(codeObjs[0])) {
```

Comment:
What's the case where the first object of the array is an array?

Reply:
This is when we have an array of multiple resources, the editor loads them at index 0 all inside it, if we have a single object as a resource then it stores it as the object only and not as array. That;s why we have to add this check

Reply:
So the parsing will be:
* If we have a list A -> loaded as [A, ]
* If we have a single resource R -> R

This looks odd.

Reply:
This is actually wrong i think i messed up somewhere with the format and parsing, I have a fix